README

```diff
@@ -6,3 +6,6 @@ http://www.apache.org/licenses/LICENSE-2.0
 
 The license file was added at revision 19f24fe on 2017-06-07, but you may
 consider that the license applies to all prior revisions as well.
+
+qtwebchannel added after revision 19f24fe on is licensed as LGPL3.

```

Comment:
`public/js/common/qwebchannel.js` added after revision 19f24fe  and is licensed as LGPL3.